builders/containerBuilder.go

```diff
@@ -0,0 +1,141 @@
+package builders
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/tfexplorer/models/generated/workloads"
+)
+
+// ContainerBuilder is a struct that can build containers
+type ContainerBuilder struct {
+	workloads.Container
+}
+
+// NewContainerBuilder creates a new container builder
+func NewContainerBuilder(nodeID string) *ContainerBuilder {
```

Comment:
This misses some mandatory fields, like flist (also check others)
Also the new function should set all optional fields to their defaults